package.json

```diff
@@ -26,7 +26,7 @@
     "commitlint": "commitlint --from=HEAD~1",
     "markdownlint": "prettier --check **/README.md docs/troubleshooting.md",
     "markdownlint-fix": "prettier --write **/README.md docs/troubleshooting.md",
-    "lint": "npm run eslint && npm run build && npm run doc && npm run commitlint && npm run markdownlint",
+    "lint": "npm run eslint && npm run build && npm run doc && npm run markdownlint",
```

Comment:
We could consider making a new target `ci-lint` or similar that skips commitlint and leave the `lint` target as is? That way, devs still runs commitlint in their local workflows per default.